### PR TITLE
Adds "Apprendre a OCaml" to the books page

### PR DESCRIPTION
**Pull request by johnwhitington**
The translation of the blurb into English was provided to me by the Authors.

### CODE REVIEW

File: site/learn/books.md [hunk 1]
```diff
@@ -224,6 +224,38 @@ runtime, interoperating with C, and pearls.
 
 ## Books in French
 
+###  Apprendre à programmer avec OCaml
+<img src="img/apprendre_ocaml_cover.png" width="180" style="float: left; margin-right: 15px; margin-bottom: 15px;"></img>
+
+*Jean-Christophe Fillâtre and Sylvain Conchon<br />
```
Comment (dbuenzli) [inline site/learn/books.md]: This should be "Filliâtre"

**Comment by pw374** [comment]
@johnwhitington please could you fix that name @dbuenzli pointed and force-push on your branch?
(This PR should automatically update and we won't have that typo in the git log.)

**Comment by johnwhitington** [comment]
Done.

**Comment by pw374** [comment]
There're still 2 commits instead of just a single one. Could you fuse them into one, otherwise I'll merge the 2 commits but it would look better if there was just one?

**Comment by johnwhitington** [comment]
I squashed them with rebase -i, and pushed it. Github seems to have updated my commit above, but let me know if I did anything wrong!

**Comment by pw374** [comment]
Sorry, it seems that it didn't work at all: you now have 4 commits according to https://github.com/johnwhitington/ocaml.org/commits/master! :(

**Comment by johnwhitington** [comment]
Ok. I'll make a new pull request from new commits, then.

**Comment by pw374** [comment]
I guess the squashing worked, but then you didn't `git push -force` to your own repo, so it asked you to pull before... Then it merged. So it ended up with 4 entries...

**Comment by Chris00** [comment]
Squashed and merged in 6add41b

**Comment by pw374** [comment]
Thank you @Chris00 :)

**Pull request closed without merging**